gensim/models/wrappers/fasttext.py

```diff
@@ -344,7 +344,7 @@ def init_ngrams(self):
         ngram_indices = []
         for i, ngram in enumerate(all_ngrams):
             ngram_hash = self.ft_hash(ngram)
-            ngram_indices.append((len(self.wv.vocab) + ngram_hash) % self.bucket)
+            ngram_indices.append((len(self.wv.vocab)) + ngram_hash % self.bucket)
```

Comment:
Removed the extra brackets.
Yes, the new operation order is current behaviour, it was a bug earlier.